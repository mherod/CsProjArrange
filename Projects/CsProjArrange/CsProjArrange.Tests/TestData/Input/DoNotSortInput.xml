<Project ToolsVersion="12.0" DefaultTargets="Build">
  <Sort>
    <C />
    <B />
    <A />
  </Sort>
  <DoNotSort>
    <F />
    <E>
      <B />
      <A />
    </E>
    <D />
  </DoNotSort>
</Project>