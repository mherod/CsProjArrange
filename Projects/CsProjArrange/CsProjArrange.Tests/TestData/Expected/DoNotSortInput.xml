<Project ToolsVersion="12.0" DefaultTargets="Build">
  <Sort>
    <A />
    <B />
    <C />
  </Sort>
  <DoNotSort>
    <F />
    <E>
      <A />
      <B />
    </E>
    <D />
  </DoNotSort>
</Project>